src/app.service.ts

```diff
@@ -0,0 +1,62 @@
+export class AppService{
```

Comment:
We need to think as different domains as different module , like Flight, User, Booking etc